plutus-tx-plugin/test/Budget/toFromData.budget.golden

```diff
@@ -1,2 +1,2 @@
-({cpu: 13616457
-| mem: 43192})
\ No newline at end of file
+({cpu: 12995457
```

Comment:
I expected a larger improvement than this for this test case. SOP is much closer to Data than Scott is, and I thought to/from Data would become way cheaper.